pkg/move/atomic_test.go

```diff
@@ -0,0 +1,55 @@
+package move
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func mockCopyFunc(isSuccessful bool) copyFunc {
+	return func(fs afero.Afero) error {
+		if isSuccessful {
+			fs.MkdirAll("/work/tmp", 0755) //nolint:errcheck
+			return nil
+		}
+
+		return errors.New("some mock error")
+	}
+}
+func TestAtomic_NoError(t *testing.T) {
```

Comment:
pls use the `t.Run`

so
```
func TestAtomic(t *testing.T) {
...
	t.Run("success -> targetFolder present", func(t *testing.T){
        ...
        })
	t.Run("fail -> targetFolder doesn't exist ", func(t *testing.T){
        ...
        })
```

Reply:
done https://github.com/Dynatrace/dynatrace-bootstrapper/pull/4/commits/a26ec38d7fa8e56ae0a5b200530ae2c4acd69a3a